### PR TITLE
Remove betauser

### PR DESCRIPTION
**Pull request by cwilliams2017**
Please Review
@rkenyon1969 
@mannyrivera2010 
@mleeBoeing 
@lsemesky 

Recommended is displayed for all Users.

### CODE REVIEW

**Comment by mannyrivera2010** [comment]
:+1:

File: ozpcenter/api/storefront/model_access.py [hunk 1]
```diff
@@ -493,8 +493,8 @@ def get_storefront_recommended(username, pre_fetch=True):
 
     profile = models.Profile.objects.get(user__username=username)
 
-    if not profile.is_beta_user():
-        return [], extra_data
+    # if not profile.is_beta_user():
```
Comment (blynch11p) [inline ozpcenter/api/storefront/model_access.py]: This won't be used any more, can you just remove the code rather than commenting out?

Comment (cwilliams2017) [reply ozpcenter/api/storefront/model_access.py]: That's true.  I will remove the code.

**Comment by blynch11p** [comment]
All users can see Recommended For You listings now.
👍

**Comment by mannyrivera2010** [comment]
:+1:

**Pull request merged**